### PR TITLE
CI

### PR DESCRIPTION
**Pull request by rammerd99**
Known issues / possible improvements:
- GUI unit tests are NOT running currently in the CI
- We could create another workflow for unit tests with windows environment
- List of generated and validated rules can be extended. Currently we are testing only with one NAT rule and one FILTER rule.

### CODE REVIEW

File: IPTables_Guide/model/parser_entries.py [hunk 1]
```diff
@@ -11,13 +11,13 @@
 }
 
 possible_tables = {
-    "-t NAT": {
-        "str_form": "-t NAT",
+    "-t nat": {
+        "str_form": "-t nat",
         "explanation": "Ezzel a kapcsolóval azt jelöljük meg, hogy a NAT(Network Address Translation) táblára fog vonatkozni az aktuális szabály.",
         "value": "NAT",
```
Comment (mscsma) [inline IPTables_Guide/model/parser_entries.py:17]: nat

File: IPTables_Guide/model/parser_entries.py [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
     },
-    "-t FILTER": {
-        "str_form": "-t FILTER",
+    "-t filter": {
+        "str_form": "-t filter",
         "explanation": "Ezzel a kapcsolóval azt jelöljük meg, hogy a FILTER táblára fog vonatkozni az aktuális szabály.",
         "value": "FILTER",
```
Comment (mscsma) [inline IPTables_Guide/model/parser_entries.py:22]: same as above

File: IPTables_Guide/model/parser_entries.py (hunk 1 above)
Comment (rammerd99) [reply IPTables_Guide/model/parser_entries.py]: done

File: IPTables_Guide/model/parser_entries.py (hunk 2 above)
Comment (rammerd99) [reply IPTables_Guide/model/parser_entries.py]: done

**Pull request merged**